### PR TITLE
Added More Actions menu to memory profiler to control displaying RSS.

### PR DESCRIPTION
**Pull request by terrylucas**
Ability to toggle viewing Resident Set Size from "More Actions" menu in Memory Profiler toolbar area.

### CODE REVIEW

File: flutter-studio/src/io/flutter/profiler/FlutterStudioProfilersView.java [hunk 1]
```diff
@@ -55,6 +63,7 @@
 import static java.awt.event.InputEvent.CTRL_DOWN_MASK;
 import static java.awt.event.InputEvent.META_DOWN_MASK;
 
+
```
Comment (jacob314) [inline flutter-studio/src/io/flutter/profiler/FlutterStudioProfilersView.java]: extra whitespace

File: flutter-studio/src/io/flutter/profiler/FlutterStudioProfilersView.java [hunk 2]
```diff
@@ -295,6 +321,42 @@ private void initializeStageUi() {
...
+    JBPopupMenu menuBar = new JBPopupMenu();
+
+    // TODO(terry): When LafIconLookup in all IDE builds use LafIconLookup.getIcon("checkmark")
+    Icon enabledCheckmark = ProfilerCheckMark;
+    Icon checkMarkSpacer = new EmptyIcon(enabledCheckmark.getIconWidth(), enabledCheckmark.getIconHeight());
+
+    JBMenuItem rssEnabledMenu = new JBMenuItem(VIEW_RSS_STATISTICS_MENU_ITEM, checkMarkSpacer);
+    rssEnabledMenu.addActionListener(event -> {
+      FlutterStudioMonitorStageView view = (FlutterStudioMonitorStageView)(this.getStageView());
+      displayRSSInformation = !displayRSSInformation;   // Toggle the state.
```
Comment (jacob314) [inline flutter-studio/src/io/flutter/profiler/FlutterStudioProfilersView.java]: Can you reuse the existing checkbox menu item class uses in the inspector menu instead?

File: flutter-studio/src/io/flutter/profiler/FlutterStudioProfilersView.java [hunk 3]
```diff
@@ -420,6 +479,11 @@ private void initializeStageUi() {
     updateStreaming();
   }
 
+  public static Icon getArrowIcon(boolean enabled) {
```
Comment (jacob314) [inline flutter-studio/src/io/flutter/profiler/FlutterStudioProfilersView.java]: add a TODO(terry) to clean up the duplicated code for the More Actions menu.

Comment (jacob314) [reply flutter-studio/src/io/flutter/profiler/FlutterStudioProfilersView.java]: duplicated between here and the Inspector.

File: flutter-studio/src/io/flutter/profiler/FlutterStudioProfilersView.java (hunk 2 above)
Comment (terrylucas) [reply flutter-studio/src/io/flutter/profiler/FlutterStudioProfilersView.java]: Yes

File: flutter-studio/src/io/flutter/profiler/FlutterStudioProfilersView.java (hunk 1 above)
Comment (terrylucas) [reply flutter-studio/src/io/flutter/profiler/FlutterStudioProfilersView.java]: Done.

**Pull request merged**